Closes: #333996
Patch from: Adrian Knoth <adi@drcomp.erfurt.thur.de>

The manpage contains several sentences with the same mistake:
the use of "then" instead of "than".


---
 logrotate.8 |   12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

Index: logrotate-3.7.1/logrotate.8
===================================================================
--- logrotate-3.7.1.orig/logrotate.8
+++ logrotate-3.7.1/logrotate.8
@@ -125,7 +125,7 @@ The next section defines the parameters 
 \fI/var/log/httpd/access.log\fR and \fI/var/log/httpd/error.log\fR.
 Each is rotated whenever it grows over 100k in size, and the old log
 files are mailed (uncompressed) to www@my.org after going through 5
-rotations, rather then being removed. The \fBsharedscripts\fR means that
+rotations, rather than being removed. The \fBsharedscripts\fR means that
 the \fBpostrotate\fR script will only be run once (after the old logs 
 have been compressed), not once for each
 log which is rotated. Note that log file names may be enclosed in
@@ -135,7 +135,7 @@ characters supported.
 
 The last section defines the parameters for all of the files in
 \fI/var/log/news\fR. Each file is rotated on a monthly basis.  This is
-considered a single rotation directive and if errors occur for more then
+considered a single rotation directive and if errors occur for more than
 one file, the log files are not compressed.
 
 Please use wildcards with caution.  If you specify *, \fBlogrotate\fR will
@@ -366,7 +366,7 @@ appear inside a log file definition. See
 \fBrotate \fIcount\fR
 Log files are rotated \fIcount\fR times before being removed or mailed to the
 address specified in a \fBmail\fR directive. If \fIcount\fR is 0, old versions
-are removed rather then rotated.
+are removed rather than rotated.
 
 .TP
 \fBsharedscripts\fR
@@ -381,7 +381,7 @@ option and implies \fBcreate\fR option.
 
 .TP
 \fBsize \fIsize\fR[\fBG\fR|\fBM\fR|\fBk\fR]
-Log files are rotated when they grow bigger then \fIsize\fR bytes. If
+Log files are rotated when they grow bigger than \fIsize\fR bytes. If
 \fIsize\fR is followed by \fIM\fR, the size if assumed to be in megabytes.
 If the \fIG\fR suffix is used, the size is in gigabytes.
 If the \fIk\fR suffix is used, the size is in kilobytes. So \fBsize 100\fR,
@@ -408,8 +408,8 @@ not commas.
 
 .TP
 \fBweekly\fR
-Log files are rotated if the current weekday is less then the weekday
-of the last rotation or if more then a week has passed since the last
+Log files are rotated if the current weekday is less than the weekday
+of the last rotation or if more than a week has passed since the last
 rotation. This is normally the same as rotating logs on the first day
 of the week, but if \fBlogrotate\fR is not being run every night a log 
 rotation will happen at the first valid opportunity.
